src/utils/streams/__tests__/split_stream.js

```diff
@@ -0,0 +1,57 @@
+import expect from 'expect.js';
+
+import {
+  createSplitStream,
+  createConcatStream,
+} from '../';
+
+async function split(stream, input) {
+  const concat = createConcatStream();
+  concat.write([]);
+  stream.pipe(concat);
+  const output = new Promise(r => concat.on('data', r));
+
+  input.forEach(i => {
+    stream.write(i);
+  });
+  stream.end();
+
+  return await output;
+}
+
+describe('splitStream', () => {
+  it('splits buffers, produces strings', async () => {
+    const output = await split(createSplitStream('&'), [Buffer.from('foo&bar')]);
```

Comment:
In other tests you use `createPromiseFromStreams`, should these also do that? (Or maybe good reason they're not?)